src/utils/formatDollars.ts

```diff
@@ -1,12 +1,12 @@
 const formatDollars = (amount: any) => {
-  if (amount) {
+  if (amount || amount === 0) {
     return amount.toLocaleString('en-US', {
       style: 'currency',
       currency: 'USD',
       minimumFractionDigits: 0
     });
   }
-  return `$${amount}`;
+  return '-';
```

Comment:
I am thinking if this `-` should be in `EstimatedLifecycleCostReview` instead of here.

`formatDollars` can be used in other places and the `-` seems specific to the table.

Reply:
I agree. I'll make this change.